package.json

```diff
@@ -118,6 +119,7 @@
         "ts-jest": "^27.0.7",
         "tsc-fancy": "^1.1.2",
         "typescript": "^4.4.4",
+        "url-polyfill": "^1.1.12",
```

Comment:
As far as I understand this is to fix the error happening in `@prezly/sdk`, right? 
Should we apply this fix there?

Reply:
Not exactly, it is used in bundler, in our case sandbox's webpack should provide polyfill. (in webpack 4 the polyfil was added out of the box, in webpack 5 you have to add it explicitly)

Reply:
It depends on the bundler, maybe some of them also provide polyfils for node packages (the URL package is inbuilt in nodejs)